src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
+```
+
+### Azure verification
+
+Get the ingress controller IP given by the cloud provider. Run
+```
+$ kubectl get svc istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].ip}"
+```
+And it will return the URL which the deployed app should reply to. Same as AWS, the wildcard `*` set as the hostname in the virtual service will send all traffic from `/healthz` path to the liveness service.
+
+Running this curl command should give you a `200` response.
+```
+curl http://<IP>/healthz -I
+```
+
+## On premise (Giantnetes)
+
+As probably you may know our [`giantnetes`](https://docs.giantswarm.io/basics/onprem-architecture/) tenant pods live inside the control plane pods. It means Giant Swarm creates a forwarding between the external ingress service to the ingress placed inside the tenant cluster. By default, the tenant clusters has a nginx ingress controller allocated out of the box. But, in case you want to use istio ingress controller you need to change the current ingress controller (remember it will affect all current traffic for that tenant cluster) or ask our team to allocate a new redirection from the parent endpoint to the Istio controller.
+
+
+Modifying the port for the ingress gateway
+```
+kubectl edit svc istio-ingressgateway -n istio-system
+----
+  - name: http2
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  - name: https
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 443
+    protocol: TCP
+    targetPort: 443
+----
+```
+
+If you run this curl command you should get a `200` response.
+```
+curl http://<LOAD_BALANCER_IP>:<PORT_TENANT_CLUSTER>/healthz -I
+```
+
+Giant Swarm encourages you to leave this liveness app up and running in order to make our monitoring happy. By default, nginx ingress controller provides a `/healthz` endpoint to check ICs are up, but Istio controller lack of this functionality. For that reason, the liveness app will give us the health signal required to have the correct observability of your istio clusters.
+
+## Troubleshooting
+
+- The pods cannot start with injected sidecars. Looking in the deployment status you see
+```
+message: 'pods "details-v1-7986ddbd99-" is forbidden: unable to validate against
+   any pod security policy: [spec.initContainers[0].securityContext.privileged:
+   Invalid value: true: Privileged containers are not allowed capabilities.add:
+   Invalid value: "NET_ADMIN": capability may not be added spec.initContainers[1].securityContext.privileged:
+```
+It means the pod security policy is not bind to the service account used by the deployment. Please review the `Preparation` section.
+
+- The request are not routed to the ingress gateway. In some cases, the default gateway is not proper configured. If istio has just been deployed, try to delete it and check now the status.
+```
+$ kubectl delete gateways.networking.istio.io istio-autogenerated-k8s-ingress -n istio-system
+```
```

Comment:
We need some explanation above this block. Or is it related to the text above? Then please change that sentence to something like `check the status again using the command below:`